libraries/AP_HAL_ChibiOS/hwdef/common/stm32l4_mcuconf.h

```diff
@@ -32,6 +32,18 @@
 #define STM32_PLLM_VALUE                    2
 #define STM32_PLLSRC                        STM32_PLLSRC_HSI16
 
+#elif STM32_HSECLK == 32768U
```

Comment:
Slightly holding my nose at the "HSE low frequency means no HSE", but I guess its ok

Reply:
yeah, slightly odd, but an awful lot simpler than changing it everywhere